formats/config/src/main/kotlin/kotlinx/serialization/config/ConfigParser.kt

```diff
@@ -21,10 +21,10 @@ public class ConfigParser(
     public inline fun <reified T : Any> parse(conf: Config): T = parse(conf, context.getContextualOrDefault(T::class))
 
     public fun <T> parse(conf: Config, deserializer: DeserializationStrategy<T>): T =
-        ConfigReader(conf).decode(deserializer)
+        ConfigDecoder(conf).decode(deserializer)
 
 
-    private abstract inner class ConfigConverter<T> : TaggedDecoder<T>() {
+    public abstract inner class ConfigConverter<T> : TaggedDecoder<T>() {
```

Comment:
This class is not intended for public use, all interactions are designed to be performed via `ConfigParser().parse`. Why do you need it to be public?